PowerPointLabs/PowerPointLabs/TextCollection/ShapesLabText.cs

```diff
@@ -69,6 +69,6 @@ internal static class ShapesLabText
         public const string FolderDialogDescription = "Select the directory that you want to use as the default.";
         public const string ErrorFolderNonEmpty = "Please select an empty folder as default saving folder.";
 
-        public const string AddShapeToolTip = "Adds a shape to Shapes Lab.";
+        public const string AddShapeToolTip = "Adds a shape to Shapes Lab. \nStart by selecting a shape.";
```

Comment:
Since you changed the original tooltip, when the button is active, it still says to "Start by selecting a shape". Perhaps we should split into a `ButtonEnabledToolTip` and `ButtonDisabledToolTip`.